portfolio/src/main/webapp/index.html

```diff
@@ -85,18 +86,34 @@ <h2> People Who inspire me </h2>
         <button onclick="addRandomGreeting()">Motivate Me! </button>
         <div style="font-size:200%;" id="greeting-container"></div>
         <form action="/data" method="POST">
-            <p>Want to leave a comment? Enter here:</p>
-            <input style="font-size:200%;" type="text" name="comment"  />
-            <br/>
-            <button>Submit</button>
+        <p>Want to leave a comment? Enter here and select your prefered language.</p>
+        <textarea  style="font-size:200%;" id="text" name="comment"></textarea>
+
+        <select style="font-size:200%;"  id="language">
+            <option value="es">English</option>
+            <option value="zh">Chinese</option>
+            <option value="es">Spanish</option>
+            <option value="fr">French</option>
+        </select>
+
+        <br/>
+        <button onclick="getPageComments();">Submit</button>
         </form>
         <p>Click here to see Page Comments!</p>
-        <button onclick="getPageComments()">Comments!</button>
+        <button onclick="getPageComments();">Comments!</button>
         <div style="font-size:200%;" id="comments-container"></div>
+        <p></p>
+      <body onload="initMap()">
```

Comment:
It's a bit weird to have a body inside of a body. The `body` element is the top level HTML element that contains all the other HTML elements. Instead, you should move this method to the original body's onload - note you can have more than one method, i.e. `<body onload = "startTimer(); initMap()">`